### PR TITLE
[RI-568] Restrict visualization of keys when size exceed a certain limit.

### PR DESCRIPTION
**Pull request by Arushi201296**
**Version**: Next version ( 1.8 )
**Ticket**: RI-568
**Description**: RedisInsight restricts viewing keys that are greater than 2 GB in size. This PR documents the same.

### CODE REVIEW

**Comment by Arushi201296** [comment]
Kindly review: @GnaneshKunal

**Review by GnaneshKunal — APPROVED** [review]

**Comment by GnaneshKunal** [comment]
Kindly review: @slemeur 

**Do not merge without explicit approval.**

This PR should be merged only after the release. We will update here to let you know when it's okay to merge and this PR should be merged along with #1037.

**Pull request merged**
